src/pages/cookie-policy/cookie-policy.constants.js

```diff
@@ -0,0 +1,113 @@
+import { styles } from './cookie-policy.styles'
+
+const sectionTitleVariant = { 
+  typography: { md: 'h4', xs: 'h5' } 
+}
+const titleVariant = {
+  typography: { md: 'h5', xs: 'h5' }
+}
+const titleWithDotVariant = {
+  typography: { xlg: 'subtitle1' }
+}
+const descriptionVariant = {
+  typography: { md: 'body1', xs: 'body2' }
+}
+
+const titleWithDotStyles = {
+  title: { 
```

Comment:
maybe you can do
const { titleWithDots, wrapper } = styles

const titleWithDotStyles= {
   titleWithDots,
   wrapper
}

Reply:
fixed